content/SCALE/SCALECLIReference/Account/CLIuser.md

```diff
@@ -34,145 +36,148 @@ You can enter commands from the main CLI prompt or from the **account** namespac
 The `create` command configures a new user account.
 
 {{< expand "Using the Create Command" "v" >}}
-
 #### Description
 
 `create` has 19 configuration properties.
-They are uid, username, group, group_create, home, home_mode, home_create, shell, full_name, email, password, password_disabled, locked, smb, sudo_commands, sudo_commands_nopasswd, sshpubkey, groups, and attributes.
-A username, full name, primary group, and password status are all required for user creation.
-For more details, see [Create Configuration Properties](#create-configuration-properties).
+They are `uid`, `username`, `group`, `group_create`, `home`, `home_mode`, `home_create`, `shell`, `full_name`, `email`, `password`, `password_disabled`, `locked`, `smb`, `sudo_commands`, `sudo_commands_nopasswd`, `sshpubkey`, `groups`, and `attributes`.
+The `username`, `full_name`, `primary_group`, and `password_status` are all required properties for user creation.
+For more details, see the table below in **Create Command Properties**.
+
+{{< expand "Create Command Properties" "v" >}}
+
+{{< include file="AccountUserProperties.md" type="page" >}}
+
+{{< /expand >}}
 
+Enter the command string with property arguments using the `=` delimiter to separate property and value, then press <kbd>Enter</kbd>. 
+For example, `uid=3000`.
 The command returns a blank line.
 
 To confirm the user is created, use [`get_user_obj`](#get_user_obj-command) or go to [**Credentials > Local Users**]({{< relref "managelocalusersscale.md" >}}) in the SCALE Web UI.
 
 #### Usage
+{{< hint type=note >}}
+This command contains the minimum required properties to successfully create a user.
+{{< /hint >}}
 
 From the CLI prompt, enter:
 
 <code>account user create username=<i>testuser</i> full_name="<i>Test User</i>" group_create=<i>true</i> password=<i>passwort1234</i></code>
 
-Press <kbd>Enter</kbd>
-
-From the **account** prompt, enter:
-
-<code>user create username=<i>testuser</i> full_name="<i>Test User</i>" group_create=<i>true</i> password=<i>passwort1234</i></code>
-
-Press <kbd>Enter</kbd>
-
-Where *testuser* is the desired username, *Test User* is the user's full name, *true* is a boolean value, and *passwort1234* is the desired password for the account.
-
-{{< hint type=note >}}
-This command contains the minimum required properties to successfully create a user.
-{{< /hint >}}
+Where:
+* *testuser* is the desired username
+* *Test User* is the user full name for the user
```

Comment:
I discussed options for describing the full name property with Tim recently, looking for something a little less term-in-the-definition. We settled on something like:

```suggestion
* *Test User* is a descriptive name for the user
```

Reply:
This is a bigger issue than just this one article, but for that field/property in general we need to come up with a more useful description for this one. "**Full name** is a full name for the user" and other versions of that don't actually tell me anything as a reader because I'm just getting the term repeated in the definition.